pkg/enclave_encrypt/encrypt.go

```diff
@@ -42,9 +46,14 @@ func (bytes *Bytes) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+type ServerMsg struct {
+	// Version of the data.
+	Version *string `json:"version,omitempty"`
+}
+
 // Message from the server with encapsulated key, quorum key signature over
 // encapsulated key and ciphertext.
-type ServerSendMsg struct {
+type ServerSendMsgV0 struct {
```

Comment:
nit: it'd be nice to move all of these types into a `types.go` or `messages.go` or `events.go` or similar so this package is easier to maintain